variables.tf

```diff
@@ -114,6 +114,17 @@ variable "vpc_subnets" {
 variable "worker_pool_id" {
   type        = string
   description = "ID of the the worker pool. It is used for the naming convention of the resources."
+  validation {
```

Comment:
I love this but it comes with the implicit requirement of using a recent Terraform version which might not be something we want to do.

Reply:
It was added in 0.13 according to a quick Google, which came out in early 2020.

Reply:
My bad. I thought it was way more recent. It is fine, then. Thanks for clarifying!